docs/source/user_guide/troubleshooting.rst

```diff
@@ -0,0 +1,39 @@
+.. _user_guide_troubleshooting:
+
+===============
+Troubleshooting
+===============
+This section explains how to resolve the most common issues encountered with ``pydpf-post``.
```

Comment:
```suggestion
This section explains how to resolve the most common issues encountered with PyPDF-Post:.
```

Reply:
There is no need to use the literal string for the module name. I see that the libary name now includes the "Py" prefix. I'll have to try to make time to edit all of your DPF files to use PyDPF-Core and PyDPF-Post.